netty/src/test/java/io/grpc/netty/WriteQueueTest.java

```diff
@@ -61,14 +62,16 @@
 @RunWith(JUnit4.class)
 public class WriteQueueTest {
 
+  private final Object lock = new Object();
+
   @Mock
   public Channel channel;
 
   @Mock
   public ChannelPromise promise;
 
-  private long flushCalledNanos;
   private long writeCalledNanos;
+  private long flushCalledNanos = writeCalledNanos + 1;
```

Comment:
I thought the purpose was to make sure `write` was called on the proper thread.  Is that not the primary purpose?